pkg/acsengine/pki.go

```diff
@@ -117,8 +117,6 @@ func createCertificate(commonName string, caCertificate *x509.Certificate, caPri
 		template.KeyUsage |= x509.KeyUsageCertSign
 		template.IsCA = isCA
 	} else if isServer {
-		extraIPs = append(extraIPs, net.ParseIP("10.0.0.1"))
```

Comment:
@ashb Why was this removed? Kubeproxy still writes an ip table rule, and kubedns relies on this.